Userland/Applications/CalendarSettings/CalendarSettingsWidget.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022-2022, Olivier De Cannière <olivier.decanniere96@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "LibGUI/ComboBox.h"
+#include "LibGUI/SettingsWindow.h"
+
+class CalendarSettingsWidget final : public GUI::SettingsWindow::Tab {
+    C_OBJECT(CalendarSettingsWidget)
+
+public:
+    virtual void apply_settings() override;
+    virtual void reset_default_values() override;
+
+private:
+    CalendarSettingsWidget();
+    Vector<String> const m_weekday_names = { "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday" };
```

Comment:
This is very similar to `long_day_names` from [AK/DateConstants.h](https://github.com/SerenityOS/serenity/blob/864c7fb9f16dadc6bc0d8c2e41f428051bc4f58e/AK/DateConstants.h#L14) (only type is different). Could we use that instead?

Reply:
Indeed. I tried using it but the `ItemListModel` template in the constructor didn't compile with an `Array`.

Reply:
oh, it’s probably because you also have specify the container type. This should work:
```cpp
GUI::ItemListModel<StringView, Array<StringView, 7>>::create(AK::long_day_names);
```

though honestly it’s a bit messy, so I don't know if we want to stick with it.

Reply:
I'll leave it as is. If someone else thinks it's better to change it, then I'll do it.